<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>메인프로그램</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="stcProcStkList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>23</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>처리대상 종목</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInStkCd">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>종목추가</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelStkCd">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>종목삭제</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblProcStkList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>521</width>
      <height>192</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <row/>
    <column>
     <property name="text">
      <string>시장구분</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>종목코드</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>처리시작일</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>저리종료일</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>최종시세시각</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnInitMnQttn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>시세 초기화</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>741</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vbChart"/>
   </widget>
   <widget class="QPushButton" name="btnRcvIdxQttn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>지수시세수신</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="edtChk">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>50</y>
      <width>201</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRcvGlobalQttn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>글로벌시세수신</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTensorFlow">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>딥러닝연결</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMnQttnOrd">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>540</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>빈분봉처리</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetFstQttn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>540</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>첫시세설정</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRcvGlobalSavedQttn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>테스트시세수신</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
